people-and-planet-ai/automl-image-classification/deploy_model.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud import aiplatform
+
+
+def create_model_endpoint(project: str, region: str, model_endpoint_name: str) -> str:
+    """Creates an AutoML model endpoint.
+
+    Args:
+        project: Google Cloud Project Id.
+        region: Location for AutoML resources.
```

Comment:
We're moving away from the term "AutoML" with regards to AI Platform. Please replace all instances of "AutoML [...]" with just "AI Platform image [...]."

Reply:
(Here and elsewhere :) .)

Reply:
Got it, just renamed everything :)